apps/api/src/metadata.ts

```diff
@@ -1,118 +1,15 @@
-            },
-            ConnectionDtoWrapper: { data: { required: true, type: () => [t['../../../libs/common/src/dtos/connection.dto'].ConnectionDto] } },
-          },
-        ],
-        [
-          import('../../../libs/common/src/dtos/provider-graph.dto'),
-          {
-            ProviderGraphDto: {
-              dsnpId: { required: true, type: () => String },
-              connections: { required: true, type: () => ({ data: { required: true, type: () => [t['../../../libs/common/src/dtos/connection.dto'].ConnectionDto] } }) },
-              graphKeyPairs: { required: false, type: () => [t['../../../libs/common/src/dtos/graph-key-pair.dto'].GraphKeyPairDto] },
-            },
-          },
-        ],
-        [
-          import('../../../libs/common/src/dtos/watch-graphs.dto'),
-          { WatchGraphsDto: { dsnpIds: { required: true, type: () => [String] }, webhookEndpoint: { required: true, type: () => String } } },
-        ],
-        [
-          import('../../../libs/common/src/dtos/graph-change-notification.dto'),
-          {
-            PersistPageUpdateDto: {
-              type: { required: true, type: () => String },
-              ownerDsnpUserId: { required: true, type: () => String },
-              schemaId: { required: true, type: () => Number },
-              pageId: { required: true, type: () => Number },
-              prevHash: { required: true, type: () => Number },
-            },
-            DeletePageUpdateDto: {
-              type: { required: true, type: () => String },
-              ownerDsnpUserId: { required: true, type: () => String },
-              schemaId: { required: true, type: () => Number },
-              pageId: { required: true, type: () => Number },
-              prevHash: { required: true, type: () => Number },
-            },
-            AddKeyUpdateDto: {
-              type: { required: true, type: () => String },
-              ownerDsnpUserId: { required: true, type: () => String },
-              prevHash: { required: true, type: () => Number },
-            },
-            GraphChangeNotificationDto: { dsnpId: { required: true, type: () => String }, update: { required: true, type: () => Object } },
-          },
-        ],
-        [import('../../../libs/common/src/dtos/graph-change-response.dto'), { GraphChangeRepsonseDto: { referenceId: { required: true, type: () => String } } }],
-      ],
-      controllers: [
-        [
-          import('./graph.controller'),
-          {
-            GraphController: {
-              getGraphs: { type: [t['../../../libs/common/src/dtos/user-graph.dto'].UserGraphDto] },
-              updateGraph: { type: t['../../../libs/common/src/dtos/graph-change-response.dto'].GraphChangeRepsonseDto },
-              watchGraphs: {},
-            },
-          },
-        ],
-        [import('./health.controller'), { HealthController: { healthz: {}, livez: {}, readyz: {} } }],
-      ],
-    },
-  };
-};
+    const t = {
```

Comment:
In account-service, `npm run format` will also update this file. I left it that way because it was easier to read, but prettier/lint doesn't seem to care about this file, so a manual `format` is needed to keep it that way.